cctp/src/utils/helper.ts

```diff
@@ -0,0 +1,30 @@
+// supported domain information - https://developers.circle.com/stablecoins/docs/supported-domains
+import { Interface } from 'ethers'
+
+export const domainToChainId: { [key: string]: string } = {
+    '0': '1', // ethereum
+    '1': '43114', // avalanche
+    '2': '10', // op
+    '3': '42161', // arb
+    '6': '8453', // base
+    '7': '137', // polygon
+  }
+  
+  export const chainIdToDomain: { [key: string]: string } = Object.entries(
+    domainToChainId,
+  ).reduce<{ [key: string]: string }>((acc, [key, value]) => {
+    acc[value] = key
+    return acc
+  }, {})
+
+  export const MINT_AND_WITHDRAW_TOPIC0 =
+  '0x1b2a7ff080b8cb6ff436ce0372e399692bbfb6d4ae5766fd8d58a7b8cc6142e6'
+
+export const MESSAGE_RECEIVE_SIG = ['0x3d12a1ff']
+export const DEX_SPAN_ABI = [
```

Comment:
why DEX_SPAN_ABI?

Reply:
change variable name, dex span is another contract onchain

Reply:
oh ok sir , i just thought of it as a way to store event and func data